pandas/core/groupby/generic.py

```diff
@@ -1373,10 +1373,6 @@ def _wrap_applied_output(self, keys, values, not_indexed_same=False):
                 # of columns
                 return self.obj._constructor_sliced(values, index=key_index)
 
-        else:
```

Comment:
Thanks @FollowTheProcess - while you're here, can you change the `elif` in line 1221 to `else`?

Reply:
Sorry yep, will do this and the xref now

Reply:
That's now done :) Thanks for helping me contribute (in an albeit microscopic way) the contributing docs made it super easy as a newbie. I hope to be able to contribute more as I learn more.